File: customlistviewsample/src/main/java/tech/thdev/customlistviewsample/data/GitHubUserResponse.java
package tech.thdev.customlistviewsample.data;

import java.util.ArrayList;

/**
 * Created by tae-hwan on 10/24/16.
 */

public class GitHubUserResponse {

    public String message;
    public String documentation_url;

    public String total_count;
    public boolean incomplete_results;
    public ArrayList<GitHubItem> items;
}

File: app/src/main/java/tech/thdev/app/data/GitHubUserResponse.kt
package tech.thdev.app.data

import com.google.gson.annotations.SerializedName

/**
 * Created by tae-hwan on 10/24/16.
 */
class GitHubUserResponse {
    @SerializedName("message")
    val message: String? = null
    @SerializedName("documentation_url")
    val documentationUrl: String? = null
    @SerializedName("total_count")
    val totalCount: String? = null
    @SerializedName("incomplete_results")
    val incompleteResults: Boolean = false
    @SerializedName("items")
    val items: List<GitHubItem>? = null
}